charts/islandis/values.dev.yaml

```diff
@@ -2302,8 +2302,7 @@ services-sessions-geoip-worker:
   env:
     GEODATADIR: '/geoip-lite/data'
     GEOTMPDIR: '/geoip-lite/data/tmp'
-    NODE_OPTIONS: '--max-old-space-size=1'
-    REDIS_URL_NODE_01: '["clustercfg.general-redis-cluster-group.5fzau3.euw1.cache.amazonaws.com:6379"]'
+    NODE_OPTIONS: '--max-old-space-size=2'
```

Comment:
I'm a bit confused, max 2mb? and how was the previous 1mb enough.
I also don't see the infra changed that would cause a change on this chart value?

Reply:
well spotted :pray:

Reply:
fixed